src/styles/buy.scss

```diff
@@ -0,0 +1,19 @@
+.buy {
+  text-align: center;
+  text-decoration: none;
+  text-transform: uppercase;
+  padding: 12px 70px;
+  color: rgba(255, 255, 255, 1);
+  background-color: $blue-accent;
```

Comment:
Ensure that the SCSS variable `$blue-accent` is defined in your stylesheets. If it's not defined, it will cause an error during the SCSS compilation.